data/json/overmap/overmap_special/specials.json

```diff
@@ -7249,5 +7249,40 @@
     "city_sizes": [ 4, 12 ],
     "occurrences": [ 0, 2 ],
     "flags": [ "CLASSIC", "LAKE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Bastion Fort",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "fort_1a" },
+      { "point": [ 1, 0, 0 ], "overmap": "fort_2a" },
+      { "point": [ 2, 0, 0 ], "overmap": "fort_3a" },
+      { "point": [ 3, 0, 0 ], "overmap": "fort_4a" },
+      { "point": [ 0, 1, 0 ], "overmap": "fort_1b" },
+      { "point": [ 1, 1, 0 ], "overmap": "fort_2b" },
+      { "point": [ 2, 1, 0 ], "overmap": "fort_3b" },
+      { "point": [ 3, 1, 0 ], "overmap": "fort_4b" },
+      { "point": [ 0, 2, 0 ], "overmap": "fort_1c" },
+      { "point": [ 1, 2, 0 ], "overmap": "fort_2c" },
+      { "point": [ 2, 2, 0 ], "overmap": "fort_3c" },
+      { "point": [ 3, 2, 0 ], "overmap": "fort_4c" },
+      { "point": [ 0, 0, 1 ], "overmap": "fort_1a_1" },
+      { "point": [ 1, 0, 1 ], "overmap": "fort_2a_1" },
+      { "point": [ 2, 0, 1 ], "overmap": "fort_3a_1" },
+      { "point": [ 3, 0, 1 ], "overmap": "fort_4a_1" },
+      { "point": [ 0, 1, 1 ], "overmap": "fort_1b_1" },
+      { "point": [ 1, 1, 1 ], "overmap": "fort_2b_1" },
+      { "point": [ 2, 1, 1 ], "overmap": "fort_3b_1" },
+      { "point": [ 3, 1, 1 ], "overmap": "fort_4b_1" },
+      { "point": [ 0, 2, 1 ], "overmap": "fort_1c_1" },
+      { "point": [ 1, 2, 1 ], "overmap": "fort_2c_1" },
+      { "point": [ 2, 2, 1 ], "overmap": "fort_3c_1" },
+      { "point": [ 3, 2, 1 ], "overmap": "fort_4c_1" }
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 10, 100 ],
+    "city_sizes": [ 3, 12 ],
+    "occurrences": [ 0, 1 ],
```

Comment:
50% chance to spawn per overmap is way too much for a fort IMO. I'd say it should be around 10% or even less.

Reply:
Good catch, the spawning was the one thing I forgot to really test as I was debugging it in. Thanks, will wait for more changes/comment and update in the morning most likely.

Reply:
So should this be occurrences 1, 10 for a proper 10% chance? I thought that was the chance of multiple ones per overmap.

Reply:
No. To set chance to spawn less than 50%, you have to add a UNIQUE flag to your overmap special definition. With this flag the "occurrences" field is calculated differently: for example, [1, 10] is calculated as "1 in 10 chance".

Reply:
Thank you, fixed. This would be good to document in doc/MAPGEN.md fixed as well.